src/board/mod.rs

```diff
@@ -579,32 +577,16 @@ impl Board {
             .count()
     }
     
-    pub fn new_chain_liberties(&self, m: Move) -> usize {
```

Comment:
If it's not used in the program, it's not useful. You'll get unused messages if you only do something in tests unless it's inside of the test mod.